db/schema.rb

```diff
@@ -96,7 +96,7 @@
 
   add_index "orders", ["user_id"], name: "index_orders_on_user_id", using: :btree
 
-  create_table "products", force: true do |t|
+  create_table "products", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.